lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -7,304 +7,380 @@
-                           }
-                       });
-                       this.topicsCollection = new TopicCollection(
-                           this.topics,
-                           {url: options.topics_url, course_id: this.course_id, parse: true}
-                       ).bootstrap();
-                       this.mainView = this.tabbedView = new ViewWithHeader({
-                           header: new HeaderView({
-                               model: new HeaderModel({
-                                   description: gettext("See all teams in your course, organized by topic. Join a team to collaborate with other learners who are interested in the same topic as you are."),
-                                   title: gettext("Teams")
-                               })
-                           }),
-                           main: new TabbedView({
-                               tabs: [{
-                                   title: gettext('My Teams'),
-                                   url: 'teams',
-                                   view: new TempTabView({text: 'This is the new Teams tab.'})
-                               }, {
-                                   title: gettext('Browse'),
-                                   url: 'browse',
-                                   view: new TopicsView({
-                                       collection: this.topicsCollection,
-                                       router: this.router
-                                   })
-                               }],
-                               router: this.router
-                           })
-                       });
-                   },
+                /**
+                 * Render the create new team form.
+                 */
+                newTeam: function (topicId) {
+                    var self = this;
+                    this.getTeamsView(topicId).done(function (teamsView) {
+                        self.mainView = new ViewWithHeader({
+                            header: new HeaderView({
+                                model: new HeaderModel({
+                                    description: gettext("Create a new team if you can't find existing teams to join, or if you would like to learn with friends you know."),
+                                    title: gettext("Create a New Team"),
+                                    breadcrumbs: [
+                                        {
+                                            title: teamsView.main.teamParams.topicName,
+                                            url: '#topics/' + teamsView.main.teamParams.topicId
+                                        }
+                                    ]
+                                })
+                            }),
+                            main: new TeamEditView({
+                                tagName: 'create-new-team',
+                                collection: this.teamsCollection,
```

Comment:
Good catch. I started down that path (so that the new team could be added to the collection) but then I didn't need it (and it can't have worked given the ``this``!). I've removed it now.